hybridse/src/udf/dynamic_lib_manager.h

```diff
@@ -27,14 +27,14 @@
 namespace hybridse {
 namespace udf {
 
-struct DynamicLibHandle {
+/*struct DynamicLibHandle {
```

Comment:
u can keep the struct, but 
1. removing ref_cnt
2. struct is not copyable
3. do `dlclose` on destructor

Reply:
raw ptr is enough